profile.h

```diff
@@ -1,24 +1,34 @@
 #ifndef PROFILE_H_
 #define PROFILE_H_
 
-constexpr kMaxVelocity = 1.0;
-constexpr kMaxAcceleration = 1.0;
+#include <math.h>
+
+struct Constraints {
```

Comment:
Why did you break this out

Reply:
Should be inside the class, since that's all thats gonna use it.

Reply:
👍